axis_backend/axis_backend/models.py

```
@@ -0,0 +1,31 @@
+    def get_queryset(self):
+        return super().get_queryset().filter(deleted_at__isnull=True)
+    
+class BaseModel(models.Model):
+    """Abstract base model with common fields for all models"""
+    id = models.CharField(primary_key=True, max_length=25, default=generate_cuid, editable=False)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    deleted_at = models.DateTimeField(null=True, blank=True)
+
+    objects = SoftDeleteManager()
+    all_objects = models.Manager()  # Includes soft-deleted records
```

Ruff (0.14.5)

Order of model's inner classes, methods, and fields does not follow the Django Style Guide: field declaration should come before manager declaration

(DJ012)